jekyll/_cci2/aws.adoc

```diff
@@ -10,7 +10,7 @@ version:
 :toc: macro
 :toc-title:
 
-Following is a step by step guide to installing CircleCI Server v{serverversion} with Terraform.
+Following is a step by step guide to installing CircleCI Server v2.x with Terraform.
```

Comment:
Should this be Server v3.x?

Reply:
Nope, this is a 2.x doc, double checking your other comments but yeah, a lot of these files weren't named clearly. Maybe tomorrow I'll take some time to refactor the 2.x docs to match new server 3 docs nomenclature